### ISSUE TITLE
Filter by Tags: List needs re-order

### ISSUE THREAD

**Issue by JunBadger** [redbadger/pride-london-app#178]
Current order of Venue and Accessibility options are placed in order from Trello ACs

This needs to be listed in alphabetical order

**Comment by robbiemccorkell** [comment]
Change has been merged. Should be good now
